### PR TITLE
documentation and My Website update

### PR DESCRIPTION
**Pull request by Choon38**
In document file prod.md include how to publish website step and update My Website.

### CODE REVIEW

File: src/i18n/translations/en.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const en = {
         my_websites: "My Websites",
         name: 'Name',
         name_cannot_be_empty: 'Name cannot be empty.',
-        name_cannot_contain_double_period: "Name can not be the '..' character.",
+        name_cannot_contain_double_period: "Name can nsot be the '..' character.",
```
Comment (vineeth-vk11) [inline src/i18n/translations/en.js]: A typo here

Comment (SnehaK253) [reply src/i18n/translations/en.js]: @vineeth-vk11 I've corrected the typo error.

**Comment by Choon38** [comment]
> [CLA assistant check](https://cla-assistant.io/HeyPuter/puter?pullRequest=161) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/HeyPuter/puter?pullRequest=161) before we can accept your contribution.**1** out of **2** committers have signed the CLA.✅ Choon38❌ sneha
> 
> **sneha** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/HeyPuter/puter?pullRequest=161) it.

I already sign CLA and also both email is same.

**Comment by jelveh** [comment]
@Choon38 there seems to be an issue with the CLA. Could you close this PR and submit another one?

**Comment by Choon38** [comment]
> @Choon38 there seems to be an issue with the CLA. Could you close this PR and submit another one?

okay

**Pull request closed without merging**